c_tests/tests/constexpr.c

```diff
@@ -0,0 +1,31 @@
+// Compiler:
+// Run-time:
+
+#include <assert.h>
```

Comment:
Can we add a comment saying what the test checks?

Reply:
Fixed in e4ccdcf.